src/Features/Core/Portable/ExternalAccess/UnitTesting/UnitTestingParsedFrameWrapper.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.StackTraceExplorer;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting
```

Comment:
📝 This needs to go in the Api namespace since it's part of one of the `IUnitTestingStackTraceServiceAccessor` signatures